examples/CMakeLists.txt

```diff
@@ -73,15 +72,17 @@ else()
             "to be turned ON - skipping")
 endif()
 
+# TODO: it looks like there is some problem with IPC implementation in Level
+# Zero on windows
 if(UMF_BUILD_GPU_EXAMPLES
    AND UMF_BUILD_LIBUMF_POOL_DISJOINT
-   AND UMF_BUILD_LEVEL_ZERO_PROVIDER)
```

Comment:
I guess `UMF_BUILD_LEVEL_ZERO_PROVIDER` shouldn't be removed here...?

Reply:
Done